Makefile

```diff
@@ -1,11 +1,11 @@
 PLATFORM_IOS = iOS Simulator,name=iPhone 11 Pro Max
 PLATFORM_MACOS = macOS
 PLATFORM_TVOS = tvOS Simulator,name=Apple TV 4K (at 1080p)
+PLATFORM_WATCHOS = watchOS Simulator,name=Apple Watch Series 4 - 44mm
 
 default: test
 
 test:
-	instruments -s devices
```

Comment:
This produces a warning on Xcode 12 and we don't really need it so let's get rid of it for now.